deployment/helm/charts/ocelot-social/templates/ingress.yaml

```diff
@@ -62,4 +62,34 @@ spec:
     regex: ^https://{{ . }}(.*)
     replacement: https://{{ $.Values.domain }}${1}
     permanent: true
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+---
+apiVersion: traefik.io/v1alpha1
+kind: Middleware
+metadata:
+  name: {{ .Release.Name }}-stripprefix
+spec:
+  stripPrefix:
+    prefixes:
+      - /imagor
```

Comment:
We're tunneling to the backend through the webapp - which is imperformant (needs to go through NodeJS runtie). We could do the same thing for `backend` service here.